extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -148,15 +148,21 @@ export class NetCoreTool {
 				child.on('exit', () => {
 					this.netCoreSdkInstalledVersion = Buffer.concat(stdoutBuffers).toString('utf8').trim();
 
-					if (semver.gte(this.netCoreSdkInstalledVersion, minSupportedNetCoreVersion)) {		// Net core version greater than or equal to minSupportedNetCoreVersion are supported for Build
-						isSupported = true;
-					} else {
-						isSupported = false;
+					try {
+						if (semver.gte(this.netCoreSdkInstalledVersion, minSupportedNetCoreVersion)) {		// Net core version greater than or equal to minSupportedNetCoreVersion are supported for Build
+							isSupported = true;
+						} else {
+							isSupported = false;
+						}
+						resolve({ stdout: this.netCoreSdkInstalledVersion });
+					} catch (err) {
+						console.log(err);
```

Comment:
Do you need to update the netCoreInstallState if this happens?

Reply:
This catch was actually moving on to reach the outer catch statement, where netCoreInstallState gets set, hence doesn't need another redundant update here.